openedx/adg/lms/webinar/admin.py

```diff
@@ -0,0 +1,25 @@
+"""
+Registering models for webinar app.
+"""
+from django.contrib import admin
+
+from .models import Webinar
+
+
+@admin.register(Webinar)
+class WebinarAdmin(admin.ModelAdmin):
```

Comment:
Please add `filter_horizontal = ('co_hosts', 'panelists', )` to have a better user experience for the admins while adding cohosts and panelists. I am suggesting all this because we do have time for this ticket.

Reply:
Also use the `raw_id_fields = ('presenter', )` to allow search for the presenter else all the users in a dropdown is a bad experience and also no way to search them as well.